packages/test/stochastic-test-utils/src/distributions/normal.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Map uniform distribution in the range [0..1) to a
+//
+// https://en.wikipedia.org/wiki/Marsaglia_polar_method
```

Comment:
Comment should move closer to code.

Reply:
There's an optimization to cache and use the 'x' value on every other call that I omitted.  Debating if it's worth it.

Reply:
I kind of doubt it matters much. That said, seems like your usage is a bit more intensive than previous cases.

Reply:
It was easier to implement than it was to explain why I didn't.  :-)